service/hook/default_response_provider.go

```diff
@@ -27,13 +27,19 @@ type SuccessRespModel struct {
 // TransformResponseModel ...
 type TransformResponseModel struct {
 	Errors                  []string                             `json:"errors,omitempty"`
+	SkippedTriggerResponses []bitriseapi.SkipAPIResponseModel    `json:"skipped,omitempty"`
```

Comment:
`json:"skipped,omitempty"` should be `json:"skipped_responses,omitempty"` - to conform to the other response names